packages/lib/modules/pool/actions/add-liquidity/modal/StakingOptions.tsx

```diff
@@ -24,13 +25,15 @@ export function StakingOptions() {
 
   const auraDisclosure = useDisclosure()
 
+  const projectConfig = getProjectConfig()
```

Comment:
Same with this, can this not just be a global const?

Reply:
added but i haven't changed it everywhere yet -> #325